integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -42,6 +47,10 @@ fn test_resharding_v3() {
         base_epoch_config_store.get_config(base_protocol_version).as_ref().clone();
     base_epoch_config.validator_selection_config.shuffle_shard_assignment_for_chunk_producers =
         false;
+    base_epoch_config.block_producer_kickout_threshold = 0;
+    base_epoch_config.chunk_producer_kickout_threshold = 0;
+    base_epoch_config.chunk_validator_only_kickout_threshold = 0;
```

Comment:
Why no kickouts? Is it temporary until all is implemented? If so can you add a todo?